net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -14,7 +14,7 @@
                     <Required>Y</Required>
                 </name>
                 <pubkey type="TextField">
```

Comment:
`Base64Field` was added in order to solve #2056 a better way, you should replace `TextField` by `Base64Field`. 
I don't know if it would then be necessary but I think that adding a <mask> </mask> may be useful to set the string length to 45 character (which is the current size of public keys)

Reply:
the change is somewhat related, but we don't need to mash up different open PRs

Reply:
Looks like it was already in master... disregard.